src/net.cpp

```diff
@@ -267,6 +267,15 @@ std::optional<CAddress> GetLocalAddrForPeer(CNode *pnode)
     return std::nullopt;
 }
 
+uint64_t GetUniqueNetworkID(const CConnman& connman, const CNode& node)
+{
+    auto local_socket_bytes = node.addrBind.GetAddrBytes();
+    return connman.GetDeterministicRandomizer(RANDOMIZER_ID_UNIQUE_NETWORK_ID)
```

Comment:
`GetDeterministicRandomizer()` does make use of `nSeed0` and `nSeed1` to construct the hasher but I think we could also just be using a `CSipHasher(0, 0)` here, since these IDs don't need to be unique per node. I left it as is to maintain current behavior.